setup.py

```diff
@@ -20,8 +20,7 @@
 ]
 
 requires = [
-    'anyjson>=0.3.3',
-    'kombu>=3.0.37',
+    'kombu>=4.0.0',
```

Comment:
Should this match the `requirements.txt` value and specify `kombu>=4.6.11`?

Reply:
Personally, I think the requirements files aren't needed. `tox` is the only thing using them and it just means that we're testing 1 version rather than supported versions :(